### PR TITLE
Update Bulgarian translation

### PR DESCRIPTION
**Pull request by rddim**

### CODE REVIEW

File: PowerEditor/installer/nativeLang/bulgarian.xml [hunk 1]
```diff
@@ -1045,9 +1045,9 @@
 		<MiscStrings>
 			<word-chars-list-tip value="Позволява добавяне на допълнителни символи към дума, чрез двойно щракане върху нея или при включена опция за търсене на &quot;Само цели думи&quot;."/>
 			<word-chars-list-warning-begin value="Бъдете внимателни: "/>
-				<!-- $INT_REPLACE$ is a place holder, don't translate it -->
+				<!-- $INT_REPLACE$ е контейнер, не се превежда -->
```
Comment (donho) [inline PowerEditor/installer/nativeLang/bulgarian.xml]: It's really no need to translate it, just leave it.

File: PowerEditor/installer/nativeLang/bulgarian.xml [hunk 2]
```diff
@@ -1045,9 +1045,9 @@
...
 			<word-chars-list-space-warning value="$INT_REPLACE$ интервал(а)"/>
-				<!-- $INT_REPLACE$ is a place holder, don't translate it -->
+				<!-- $INT_REPLACE$ е контейнер, не се превежда -->
```
Comment (donho) [inline PowerEditor/installer/nativeLang/bulgarian.xml]: No need to be translated.

File: PowerEditor/installer/nativeLang/bulgarian.xml (hunk 1 above)
Comment (rddim) [reply PowerEditor/installer/nativeLang/bulgarian.xml]: Corrected

File: PowerEditor/installer/nativeLang/bulgarian.xml (hunk 2 above)
Comment (rddim) [reply PowerEditor/installer/nativeLang/bulgarian.xml]: Corrected

**Pull request closed without merging**